lib/read_concern.js

```diff
@@ -34,6 +34,22 @@ class ReadConcern {
 
     return new ReadConcern(options.level);
   }
+
+  static get LEVEL_MAJORITY() {
```

Comment:
nit: personally I'd say drop the `LEVEL_` prefix. Very nice idea adding these statics!